lib/json_schemer/schema/base.rb

```diff
@@ -54,10 +54,29 @@ def validate(data)
 
     protected
 
+      def discriminate(instance, field_name)
```

Comment:
wrong number of args here. should add a test like this to check that using OA attributes in other schemas doesn't cause them to break, e.g:
```ruby
  def test_do_unknown_properties_break_other_schemas
    schema = JSONSchemer.schema(
      'discriminator' => 'yup', # or any truthy value
      "oneOf" => [{'type' => 'object'}],
    )
    assert schema.valid?('a' => 1)
  end
```

Reply:
@dleavitt Thanks again. I have moved the method out of Base, because it  does not make sense there. 👍